### DISCUSSION TITLE
Function reference

### DISCUSSION THREAD

**Discussion by sam701** [tweag/nickel#804] [General]
The language is very promising. But where can I find reference of available functions? The official site https://nickel-lang.org/ does not contain anything like that? Am I missing something?

**Comment by yannham** [comment]
Ah, a basic documentation can be extracted from the stdlib, but couldn't be displayed on the website yet, waiting for the version 0.2.0. But it has happened since, so we should put this documentation online. Thanks for the reminder :slightly_smiling_face: I will update this discussion once this is done.

**Reply by sam701** [reply]
Great, looking forward. 👍
